app/src/main/java/org/feature/fox/coffee_counter/data/repository/UserRepositoryInt.kt

```diff
@@ -35,6 +36,12 @@ interface UserRepositoryInt {
 
     suspend fun deleteImageDb(image: Image)
 
+    suspend fun getPurchaseListOfUserDb(userId: String): List<Purchase>
+
+    suspend fun getAchievementListOfUserDb(userId: String): List<Achievement>
+
+    suspend fun insertAchievementDb(achievement: Achievement)
```

Comment:
Same here with the sorting, doesn't need to be in the implementation, but in the interface it's really nice to have.